docs/kubernetes/fsgroup-workaround.md

```diff
@@ -0,0 +1,97 @@
+# CSI driver FsGroup Workaround User Guide
+
+>**Attention:** This workaround to apply fsgroup to volumes provisioned by filestore driver is no longer needed when using `stable-master` overlay driver manifest bundle.
+
+>**Attention:** `CSIVolumeFSGroupPolicy` is a Kubernetes feature which is Beta is 1.20+, Alpha(1.19).
+
```

Comment:
mention somewhere that the feature gate needs to be enabled on the cluster for the stable-master overlay to work, otherwise you need to use the workaround? (I think that's correct)

Reply:
Done